app/assets/main/react_components/context/Footprint/TextsContext.js

```diff
@@ -0,0 +1,30 @@
+import React, { useContext } from "react";
+
+const TextsContext = React.createContext(undefined);
+export const useTexts = () => {
+    const context = useContext(TextsContext)
+    if(!context)
+        throw new Error('TextsContext does not exist.')
+    return context
+}
+
+export const TextsProvider = ({ children, commonText, lifestyleFootprintsText, modelText, registrationsText, slug }) => {
+    
+    const texts = {
+        commonText: JSON.parse(commonText),
+        lifestyleFootprintsText: JSON.parse(lifestyleFootprintsText),
+        modelText: JSON.parse(modelText),
+        registrationsText: JSON.parse(registrationsText)
+    };
+
+    if(texts.registrationsText.accept_policies === undefined)
```

Comment:
```suggestion
    if( texts.registrationsText.accept_policies === undefined)
```
Lintern kanske fixar detta?

Reply:
@teolenn kodformateraren Prettier kommer fixa!